docs/using-the-jdbc-driver/using-plugins/UsingTheReadOnlyEndpointConnectionPlugin.md

```diff
@@ -0,0 +1,12 @@
+# Read Only Endpoint Connection Plugin
+
+When making connections to the reader cluster endpoint of an Aurora database, each connection is made to a random available reader instance. This may cause errors if operations depend on making connections to the same instance. For example, to execute a query timeout with the MySQL Connector/J 8.0 Driver, a new internal connection will be made to kill the initial query. If the reader endpoint is used, the kill query may fail to cancel the initial query because the queries will be made to different hosts.
```

Comment:
Can we make clear distinction between reader instance endpoint and reader cluster endpoints?

Reply:
Can we draw more connection on how the plugin resolves the problem?